cmd/containers-storage/main.go

```diff
@@ -32,6 +33,7 @@ func main() {
 	if reexec.Init() {
 		return
 	}
+	unshare.MaybeReexecUsingUserNamespace(false)
```

Comment:
I think this should be a separate commit. I am unable to figure out if this is related to logic change.

Reply:
Dropped it.